documentation/modules/exploit/linux/http/oracle_ebs_rce_cve_2022_21587.md

```diff
@@ -0,0 +1,96 @@
+## Vulnerable Application
+
+This module exploits CVE-2022-21587, an arbitrary file upload vulnerability in Oracle Web Applications Desktop Integrator as shipped with Oracle E-Business Suite (EBS) versions 12.2.3 through to 12.2.11.
```

Comment:
This file should likely be linted with `tools/dev/msftidy_docs.rb documentation/modules/exploit/linux/http/oracle_ebs_rce_cve_2022_21587.md`. There are some excessively long lines here and we are missing some of the expected titles and formatting.

Reply:
All msftidy_docs warnings resolved via [d1463df](https://github.com/rapid7/metasploit-framework/pull/17624/commits/d1463df3ccf55e540399ae2efe56a3869362f376)

Reply:
Thanks @sfewer-r7 formatting looks great! Marking approved 👍